src/org/zaproxy/zap/extension/advreport/resources/help/contents/advreport.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Customizable HTML Report
+</TITLE>
+</HEAD>
+<BODY BGCOLOR="#ffffff">
+
+<H1>Customizable HTML Report</H1>
+The advrepot extension allows you to control the information that goes into the ZAP scanning report.
```

Comment:
That's fine by me (although for the package is should be all lower case, and camel case for the class name).

Any other opinions about the change?